server/graphql/v1/inputTypes.js

```diff
@@ -187,6 +187,7 @@ export const CollectiveInputType = new GraphQLInputObjectType({
     firstName: { type: GraphQLString },
     lastName: { type: GraphQLString },
     isIncognito: { type: GraphQLBoolean },
+    isDeletable: { type: GraphQLBoolean },
```

Comment:
I see that you're following the example of `isIncognito`, which was not a great model in the first place, I'm sad we accepted that change when it was introduced.

I think we should better filter these properties before submitting to GraphQL.

Reply:
I'm not sure If I understood what you're saying. Could you give me more details? Is the problem that the field being "calculated" on the GraphQL?
![image](https://user-images.githubusercontent.com/294406/69200327-8dd73280-0b19-11ea-9eb8-e736c08d523a.png)

Reply:
Yes, it's a dynamic "calculated" field, there is no reason to add it in the Input because we will never want to update it. I think you're adding it here to prevent an error, but I think we should not send it in the first place.

Reply:
Yes, I added here because the e2e tests for the front were broking asking for this field on the schema.